test_runner/batch_others/test_wal_acceptor_async.py

```diff
@@ -0,0 +1,166 @@
+import asyncio
+import aiopg
+import random
+
+from fixtures.zenith_fixtures import WalAcceptor, WalAcceptorFactory, ZenithPageserver, PostgresFactory, Postgres
+from typing import List
+
+pytest_plugins = ("fixtures.zenith_fixtures")
+
+
+class BankClient(object):
+    def __init__(self, conn: aiopg.Connection, n_accounts=None, init_amount=None):
+        self.conn: aiopg.Connection = conn
+        self.n_accounts = n_accounts
+        self.init_amount = init_amount
+
+    async def initdb(self):
+        async with self.conn.cursor() as cur:
+            await cur.execute('DROP TABLE IF EXISTS bank_accs')
+            await cur.execute('CREATE TABLE bank_accs(uid int primary key, amount int)')
+            await cur.execute(
+                '''
+            INSERT INTO bank_accs
+            SELECT *, %s FROM generate_series(0, %s)
+            ''', (
+                    self.init_amount,
+                    self.n_accounts - 1,
+                ))
+
+            await cur.execute('CREATE TABLE bank_log(from_uid int, to_uid int, amount int)')
```

Comment:
Shouldn't we also drop this table if it exists (or `create ... if not exists`)?

Reply:
Nice catch, will fix soon